docs/application/tizen-studio/setup/openjdk.md

```diff
@@ -1,5 +1,8 @@
 # OpenJDK 12 and OpenJFX Installation Guide
 
+> [!NOTE]
+> It is highly recommended to use the latest version of Tizen Studio. Since Tizen Studio 3.7, the JDK installation is bundled with Tizen Studio installation. There is no need to explicitly install JDK or set the variable paths. The following procedure of installing the JDK and setting the JDK variables is applicable for Tizen Studio 3.6 or lower. 
```

Comment:
the JDK installation is bundled with Tizen Studio installation

I think this should be "JDK is bundled with Tizen Studio"

@manishtr can you review this note?

@mijin85cho, should we retain the steps for OpenJDK installation, it applies for 3.6 or earlier versions.

Reply:
Yes it looks right to me, thanks for the update

Reply:
Thanks for checking, @annie-samsung 

- As you said, Tizen Studio before 3.7 is old version. Considering that, we already have a note at the [JDK description](https://github.com/Samsung/tizen-docs/blob/4444a0d94ea86f592e0054ad2d5757ad4e7f7193/docs/application/tizen-studio/setup/prerequisites.md#java-development-kit-jdk-requirements). However it seems it wasn't enough, as we got the issue. So adding another note at here looks good to me.


- In this page, there are other several notes which are in old style. How about updating those notes also, as for now old & new style notes are mixed in this page?

- I agree your below suggestion. This update is also needed
   >Update the sentence of list item 5 :
   >5. Enter JAVA_HOME in the variable name field and the JDK directory path to the bin directory, for example, C:\Users\user\Desktop\jdk-12.0.2\bin, in the Variable value field.
   >
   >Add a note after list item 10.
   >Note :
   >If a different Java version is installed and preferred on the system, the ability to deploy or debug may be affected.

Reply:
@nawab-samsung, as discussed earlier and on chat, please check this bit https://www.youtube.com/watch?v=zzZiQmNcyq8&t=41s